client/components/admin/UserList.jsx

```diff
@@ -0,0 +1,195 @@
+    this.props.actions.setSelectedUser(userId);
+    this.props.actions.displaySelectedUser();
+  }
+  deleteUser() {
+    const userId = this.state.id;
+    this.props.actions.deleteUser(userId)
+    .then(() => toastr.success('User Successfully Deleted'))
+    .catch(() => {
+      this.props.addFlashMessage({
+        type: 'error',
+        text: 'Unable to delete user' });
+      toastr.error(
+        'Unable to delete user');
+    });
+    this.setState({ id: 0 });
+  }
+
+  handlePageClick(data) {
+    const selected = data.selected;
+    const offset = Math.ceil(selected * this.state.limit);
+
+    this.setState({ offset }, () => {
+      this.props.actions.loadUsers(this.state.limit, offset);
+    });
+  }
+
+  renderAlert(event) {
+    e.ventpreventDefault();
+    let id = this.state.id;
+    id = event.target.id;
+    this.setState({ show: true, id });
+    this.props.swal({
+      title: 'Warning!',
+      text: 'Are you sure?',
+      type: 'info',
+      showCancelButton: true,
+      onConfirm: this.deleteUser,
+      onCancel: this.props.close,
+    });
+  }
+
+  render() {
+    return (
+      <div>
+      {this
+        .props
+        .allUsers
+        .map(user => <div id="card-alert" className="card"
+        key={user.id}>
+          <div className="card-content teal-text transparent80">
+            {user.name} - {user.email}
+          </div>
+          <div className="fixed-action-btn horizontal click-to-toggle edit">
+            <a className="btn-floating teal tooltipped"
+              data-position="top" data-delay="50"
+              data-tooltip="click to view more"
+              >
+              <i className="material-icons">more_vert</i>
+            </a>
+            <ul>
+            <li onClick={this.viewUser} className="editDoc">
+              <a
+              className="btn-floating teal tooltipped"
+              data-position="bottom" data-delay="50"
+              data-tooltip="edit document">
+                <i id={user.id} className="material-icons">view_list</i>
+              </a>
+            </li>
+              <li onClick={this.editUser} className="editDoc">
+                <a
+                className="btn-floating teal tooltipped"
+                data-position="bottom" data-delay="50"
+                data-tooltip="edit document">
+                  <i id={user.id} className="material-icons">mode_edit</i>
+                </a>
+              </li>
+              <li onClick={this.renderAlert}>
+                <a className="btn-floating red accent-4 tooltipped"
+                  data-position="bottom" data-delay="50"
+                  data-tooltip="delete document">
+                  <i id={user.id} className="material-icons">delete</i>
+                </a>
+              </li>
+            </ul>
+          </div>
+        </div>)}
+        <div className="row">
+          <div className="col s7">
+            <ReactPaginate previousLabel={'previous'}
```

Comment:
You can adjust the indentation (2 spaces) and place each property on a different line.